elrond-wasm/src/types/managed/managed_vec.rs

```diff
@@ -127,6 +127,12 @@ where
         }
     }
 
+    pub fn set(&mut self, index: usize, item: &T) -> bool {
+        let byte_index = index * T::PAYLOAD_SIZE;
+        item.to_byte_writer(|slice| self.buffer.set_slice(byte_index, slice))
+            .is_ok()
+    }
```

Comment:
This function should return a result with a proper error, since the `bool` is very likely to be ignored in production code.

Reply:
Result<(), BadIndex>, or something of the sort is just as efficient as bool and much easier to read